docs/platforms/aws-lambda/nodejs/index.md

```diff
@@ -0,0 +1,465 @@
+# AWS Lambda: Node.js
+
+AWS Lambda is a serverless compute service that lets you run code without provisioning or managing servers.  You can measure Metrics, Traces and Logs of AWS Lambda via Serverless Console.  
+
+# Tracing
+
+If you use Serverless Console's AWS Lambda Extension for Node.js, it will automatically trace your AWS Lambda invocations, providing a timeline rich with information we curated to help you assess, optimize and troubleshoot, as well as trace services and data that triggered your AWS Lambda functions.
+
+Before the handler of your AWS Lambda function runs, our Extension loads our SDK and performs the auto-instrumentation logic.  This is what it looks like:
+
+```javascript
+import serverless from 'serverless-sdk-aws-lambda'
+
+serverless.instrument(params)
+
+// Your handler is called next...
+```
+
+Every AWS Lambda function invocation instrumented with our Extension generates a Trace.  This Trace contains the following Spans, some of which are optional depending on the modules you use within your code.  We put great effort into enriching this Trace with Tags useful for debugging across code, AWS Services and more.
+
+Here is a table of contents of Spans we currently capture, in an example hierarchical format:
+
+* **aws.lambda**
+  * **aws.lambda.initialization**
+  * **aws.lambda.invocation**
+    * **express**
+      * **node.https**
+      * **node.http**
+      * **aws.sdk**
```

Comment:
If I understand correctly under one `aws.lambda` we can get multiple `aws.lambda.invocation` spans (and depending on decision on how we handled initialization after crashes, we could also have multiple `aws.lambda.intialization`)

So the flow could be as:

```
aws.lambda
  aws.lambda.initialization
  aws.lambda.invocation
    express
      ...
  aws.lambda.invocation
    express
      ...
  aws.lambda.initialization
  aws.lambda.invocation
    express
      ...
  aws.lambda.invocation
    express
      ...
```

Is my understanding correct?

Or do we want to create different `aws.lambda` span parent for each invocation?